docs/bigquery-gettingstarted.md

```diff
@@ -0,0 +1,150 @@
+    * These tables are very large (47GB as of Aug 2017)
+* yyyy_mm_dd_pages_mobile and yyyy_mm_dd_requests_mobile:
+    * Same as above, buf for mobile pages.
+ 
+### httparchive.har Tables:
+* yyyy_mm_dd_chrome_pages:
+    * HAR extract for each page url
+    * Table contains a url and a JSON-encoded HAR file for the document.
+    * These tables are large (~3GB as of Aug 2017)
+* yyyy_mm_dd_chrome_requests:
+    * HAR extract for each resource
+    * Table contains a document url, resource url and a JSON-encoded HAR extract for each resource
+    * These tables are very large (~168GB as of Aug 2017)
+* yyyy_mm_dd_chrome_requests_bodies:
+    * HAR extract containing response bodies for each request
+    * Table contains a document url, resource url and a JSON-encoded HAR extract containing the first 2MB of each response body.   
+    * payloads are truncated a 2MB, and there is a column to indicate whether they payload was truncated.
+    * These tables are very large (~760GB as of Aug 2017)
+* yyyy_mm_dd_chrome_lighthouse:
+    * Results from a [Lighthouse](https://developers.google.com/web/tools/lighthouse/) audit of a page
+    * Table contains a url, an a JSON-encoded copy of the lighthouse report.
+* yyyy_mm_dd_android_pages yyyy_mm_dd_android_requests yyyy_mm_dd_android_requests_bodies yyyy_mm_dd_android_lighthouse:
+    * Same as chrome tables above, but for mobile pages captured by an emulated mobile device.
+
+
+## Useful Links for BigQuery SQL Reference
+BigQuery supports two SQL dialects: standard SQL and legacy SQL.  Legacy SQL is a non-standard SQL dialect that BigQuery started out using.   Standard SQL is a a SQL2011 compliant dialect that has many more features and functions.  Standard SQL is the preferred dialect for querying data via BigQuery, but both are supported.
+
+[Documentation of Standard SQL](https://cloud.google.com/bigquery/docs/reference/standard-sql/)
+
+[Documentation for Legacy SQL](https://cloud.google.com/bigquery/docs/reference/legacy-sql)
+
+If you have existing Legacy SQL that you are trying to migrate to Standard SQL, then you may want to read the ![migration guide.](https://cloud.google.com/bigquery/docs/reference/standard-sql/migrating-from-legacy-sql)
+
+When you are ready to start writing queries, make sure that the SQL dialect option selected matches what you are writing your query in.   The current default is Legacy SQL, but de-selecting this will switch to Standard SQL
+![SQL Dialect](images/standard_or_legacy_sql.jpg)
+
+The [HTTP Archive Discuss section](https://discuss.httparchive.org/) has lots of useful examples and discussion on how to analyze this data.
+
+## Some Example Queries to Get Started Exploring the Data
+Now that you are all set up, let's run a query!  To start off, try running a simple aggregate query that tells you how many URLs are contained in the latest HTTP Archive data.
+
+```
+SELECT COUNT(*)
+FROM `httparchive.runs.2017_09_01_pages`
```

Comment:
Since you mention "the latest data" maybe change this to the `_latest` table? Otherwise the docs will feel out of date. Maybe update the screenshot to match the query as well?

Reply:
The reason why I haven't been using the _latest tables is because they are Legacy SQL views and won't work with Standard SQL.   Should I just remove the parts about the latest tables (or is there a way to make those views compatible w/ Standard SQL?

![image](https://user-images.githubusercontent.com/7459458/30762855-aa173b14-9fb1-11e7-9dfc-f9aeb2b21ff6.png)

Reply:
Changing the earlier section to avoid the confusion:

```
The `latest_` views contain the most recent data, but can only be used with Legacy SQL (see below for difference between Legacy and Standard SQL)
```